tools/create_thumb.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -x
```

Comment:
if `set -x` I would recommend to ensure that PS4 is set.  Quite often in my case, not setting it explicitly leads to side effects.  My favorite is smth like
```
$> bash -c "( PS4='[$?][$(date +%DT%T)]> '; set -x; echo 1; echo 2; )"   
[0][06/26/20T11:59:50]> echo 1
1
[0][06/26/20T11:59:50]> echo 2
2
```
so I could see when was it ran and either previous command failed.

note: ideally PRs shouldn't mix in unrelated changes -- this script could have gone to a new branch/separate PR and could have been merged without waiting for JS changes.

Reply:
Oh~ that's a cool trick. I didn't know there was `set -x` specific PS configuration! You can add $LINENO!

```
The PS4 defined below in the ps4.sh has the following two codes:

$0 – indicates the name of script
$LINENO – displays the current line number within the script
```

What sort of side-effects have you ran into?

And yes, I wouldn't have mixed unrelated changes if I was doing this a bit more professionally.. :) I will move them out to the new -thumbnail you just setup.

Reply:
> What sort of side-effects have you ran into?

I think whenever my zsh had it set to some non-bash digestable value but then I was sourcing some bash script, it lead to crashes from eval'ing of that PS4.  So I just usually now make it explicit